.github/workflows/integration-tests.yaml

```
@@ -0,0 +1,44 @@
+name Integration Tests
```

actionlint (1.7.7)

workflow is scalar node but mapping node is expected

(syntax-check)